### DISCUSSION TITLE
8x stack for crafting resources isn't enough

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#10011] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Self-explanatory, _every_ sub storage cabinet eventually looks like this as early as the second biome:

![image](https://user-images.githubusercontent.com/73229309/194439338-88a855a7-cdcc-4de7-a8a4-3a15e62a19a4.png)

Storage containers singlehandedly exist to demultiply the storage room for materials, as a result we end up with several resources packed inside resource containers which make it impossible to visulalize resources at a glance. 

I don't think there's an issue with increasing resource stacks to 32x for all crafting resources, it's a number that's high enough that you don't need storage containers and low enough to discourage hoarding. 

Lower stack limits can be kept for crates or toolbelts using `maxstacksize` in their respective properties

**Comment by HeatedWafflez** [comment]
true... true... i agree with your statement
(i unironically do though this needs to change)

**Comment by ubertpendragon** [comment]
Posting public campaign cabinets every day:

![image](https://user-images.githubusercontent.com/73229309/194619990-01e212dc-0a6b-4a6c-bc24-b334595f42ad.png)

**Comment by Ek-Videogames** [comment]
_Uber & making inventory balance suggestions that actually make sense!? Pinch me I must be dreaming!_

+1 Absolutely, especially when the 'resource storage' on even the largest vanilla subs is still no more than just a single large steel cabinet.

**Comment by ubertpendragon** [comment]
![image](https://user-images.githubusercontent.com/73229309/194710461-04bcd20e-6818-48e1-8dfe-d42bcf00f475.png)
![image](https://user-images.githubusercontent.com/73229309/194712248-fda4ebac-6c0e-440c-a0f9-486425da5cff.png)

**Comment by ubertpendragon** [comment]
Day 4
![image](https://user-images.githubusercontent.com/73229309/194780312-0d559330-0a11-47e4-9fdd-1c96c8604daf.png)

**Comment by ubertpendragon** [comment]
Day 5
![image](https://user-images.githubusercontent.com/73229309/194875397-28dacd16-9d10-48ea-8179-a1d9d5d20ea0.png)

**Comment by ubertpendragon** [comment]
My last, you get the point
![image](https://user-images.githubusercontent.com/73229309/196002505-623c604f-ba90-4595-952f-2100934ff3df.png)
